db/migrate/20190711143928_devise_create_users.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ""
+      t.string :encrypted_password, null: false, default: ""
+
+      ## Recoverable
+      # t.string   :reset_password_token
+      t.datetime :reset_password_sent_at
+
+      # Please note - we are keeping these fields in for reference.
+      # When we come to implement these features we will need to create a new migration that includes these fields, rather than uncommenting these.
+
+      ## Trackable
+      # t.integer  :sign_in_count, default: 0, null: false
+      # t.datetime :current_sign_in_at
+      # t.datetime :last_sign_in_at
+      # t.string   :current_sign_in_ip
+      # t.string   :last_sign_in_ip
+
+      ## Confirmable
+      # t.string   :confirmation_token
+      # t.datetime :confirmed_at
+      # t.datetime :confirmation_sent_at
+      # t.string   :unconfirmed_email # Only if using reconfirmable
+
+      ## Lockable
+      # t.integer  :failed_attempts, default: 0, null: false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
```

Comment:
As discussed with @Rosa-Fox offline, it's cool to hang to all this to save time in the future, but if these are uncommented in the same file with the expectation they will be applied in the next migration I believe Rails will ignore them as it will think it's already run the file.

Reply:
Thanks @brenetic. Added a comment to the migration to explain that we have kept the fields commented in there for reference and that a new migration will need to be created when we choose to implement them.